packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -319,12 +324,15 @@ class DefaultTextEditingShortcuts extends Shortcuts {
     const SingleActivator(LogicalKeyboardKey.keyC, meta: true): CopySelectionTextIntent.copy,
     const SingleActivator(LogicalKeyboardKey.keyV, meta: true): const PasteTextIntent(SelectionChangedCause.keyboard),
     const SingleActivator(LogicalKeyboardKey.keyA, meta: true): const SelectAllTextIntent(SelectionChangedCause.keyboard),
+    const SingleActivator(LogicalKeyboardKey.keyZ, meta: true): const UndoTextIntent(SelectionChangedCause.keyboard),
```

Comment:
will this fire multiple times when user press and hold?

Reply:
Yes, by default repeat events count as SingleActivator.

Reply:
I just confirmed that holding it down did repeat undo/redo, which seems to be the correct behavior 👍 .